archetypes/project-bare/src/main/resources/archetype-resources/pom.xml

```diff
@@ -237,5 +237,17 @@ Bundle-SymbolicName: ${project.groupId}.${project.artifactId}
                 </plugin>
             </plugins>
         </pluginManagement>
+
```

Comment:
Is there a reason that this needs to be in `<plugins>` not `<pluginManagement>`? It looks like it has to be defined in the runnable project anyway.

Reply:
Yes, it's explained in the comment. If the comment isn't clear enough I'm up for suggestions to improve it.